antenna/ext/s3/connection.py

```diff
@@ -17,6 +17,17 @@
 logger = logging.getLogger(__name__)
 
 
+def wait_times_save():
+    """Wait time generator for failed save attempts
+
+    This waits 2 seconds between failed save attempts for 5 iterations and then
+    gives up.
+
+    """
+    for i in [2, 2, 2, 2, 2]:
+        yield i
```

Comment:
This is a much nicer construct for dealing with "what's the next wait time" as well as "have I exceeded the maximum number of retry attempts". Previously, the retry attempts were hard-coded and we'd pass in a `max_attempts` argument. That felt like a clumsier API.